File: codeforces/2038-N.cpp
#include <iostream>
#include <vector>
#include <algorithm>
using namespace std ;

void solve(){
    ios_base::sync_with_stdio(false);
    int n  ;
    cin >> n;
    for (int i ; i<n ; i++ ){
        string s ;
        getline(cin ,s, '');
        cin >> s ;
        if (s.find("<") ) {
            vector<string> v ;
            v = s.s;
         }
        if (s.find(">") ) {
            int x  = s.find(">");
        }
        if (s.find("=") ) {
            int x  = s.find("=");
            
        }
        
    }
    
}


int main()
{
    solve();
    return 0;
}

File: codeforces/2038-N.py
def solve(s):
    a = int(s[0])
    op = s[1]
    b = int(s[2])

    if (op == '<' and a < b) or (op == '>' and a > b) or (op == '=' and a == b):
        return s

    if a < b:
        return f"{a}<{b}"
    elif a == b:
        return f"{a}={b}"
    else:
        return f"{a}>{b}"


for _ in range(int(input())):
    s = input()
    print(solve(s))
